backend/src/main/java/com/crm/model/db/RoleEntity.java

```diff
@@ -13,12 +9,13 @@
 import javax.persistence.Table;
 
 @Entity
-@Table(name = "roles")
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
 @Getter
 @Setter
+@EqualsAndHashCode
```

Comment:
To samo :D

Reply:
To nie zadziała z Builderem, zostawiłem jak było